### PR TITLE
89 add riconv++

### PR DESCRIPTION
**Pull request by annprzy**

### CODE REVIEW

File: scripts/train_contiguous.py [hunk 1]
```diff
@@ -66,8 +67,12 @@ def seed_worker(worker_id):
         for idx, (batch, labels) in enumerate(train_dataloader):
             labels = labels.to(device=device)
             batch = batch.to(device=device)
-            labels_hat = model(batch)
-            loss = criterion(labels_hat, labels) / accum_iter
+            try:
+                labels_hat = model(batch)
+            except:
+                continue
```
Comment (jkarolczak) [inline scripts/train_contiguous.py:73]: do we really need such a try-except? Is the model not capable of handling mini-batches of different sizes?

File: scripts/train_contiguous.py [hunk 2]
```diff
@@ -55,7 +55,8 @@ def seed_worker(worker_id):
 
     optimizer = torch.optim.AdamW(model.parameters(), lr=float(cfg["lr"]), weight_decay=float(cfg["weight_decay"]))
     scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=12, gamma=0.1)
-    criterion = torch.nn.CrossEntropyLoss()
+    # criterion = torch.nn.CrossEntropyLoss()
+    criterion = torch.nn.NLLLoss()
```
Comment (jkarolczak) [inline scripts/train_contiguous.py:59]: We can revert changes using git. I think we can get rid of this comment

File: scripts/train_contiguous.py (hunk 1 above)
Comment (annprzy) [reply scripts/train_contiguous.py:73]: it is necessary because RiConv can't handle batches of size 1 (due to batch normalization).

File: scripts/train_contiguous.py (hunk 2 above)
Comment (annprzy) [reply scripts/train_contiguous.py:59]: Actually I'm not sure if we want to leave the criterion "hard-coded", as this train script is used with pointnet etc, not only with riconv.  Please, tell me what you think and whether we should modify it in this issue or create a new one for that purpose.

File: README.md [hunk 3]
```diff
@@ -76,3 +76,12 @@ We provide a docker setup with all dependencies and runtime configuration. To st
...
 - Surface (neighbourhood=22) + uniform selection (max_blob_size=2000,
   method=max) [[download](https://ligands.blob.core.windows.net/ligands/blobs_shell_22_uniform_2000_max.tar.gz)]
+
+
+## RiConv
+
+To use RiConv, install the Pointnet++ cuda operation library by running the following commands:
+```
+cd src/models/contiguous/riconv2/pointnet2/
+python setup.py install
```
[the remark this answers is not in the document]
Comment (annprzy) [reply README.md:86]: That would be great, here is the issue: #100

**Review by wtaisner — APPROVED** [review]
